src/client/components/app/Transactions/EnableSpigotTx.tsx

```diff
@@ -193,10 +193,15 @@ export const EnableSpigotTx: FC<EnableSpigotTxProps> = (props) => {
     },
   ];
 
-  const renderFuncSelectors = () =>
-    revFuncDisplayConfigs.map(({ header, byteCode, options, type, onChange, onByteChange }) =>
+  const renderFuncSelectors = () => {
+    // console.log('func selector', selectedContractFunctions[revenueContractAddy]);
+    return revFuncDisplayConfigs.map(({ header, byteCode, options, type, onChange, onByteChange }) =>
       // if no ABI, input bytecode manually
-      isValidAddress(revenueContractAddy) && contractABI ? (
+
+      isValidAddress(revenueContractAddy) &&
+      contractABI &&
+      selectedContractFunctions[revenueContractAddy] !== undefined &&
```

Comment:
can u combine these two lines into `!!selectedContractFunctions[revenueContractAddy]?.length`. should be the same thing